logging_demo/CMakeLists.txt

```diff
@@ -0,0 +1,114 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(logging_demo)
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra)
```

Comment:
Nit: Adding a comment here explaining why we cant use pedantic like the other repos will be useful for book keeping